tests/unit/amo/components/TestRatingManager.js

```diff
@@ -16,73 +9,76 @@ import {
   ADDON_TYPE_STATIC_THEME,
   ADDON_TYPE_THEME,
 } from 'core/constants';
-import I18nProvider from 'core/i18n/Provider';
 import { initialApiState } from 'core/reducers/api';
 import * as reviewsApi from 'amo/api/reviews';
-import createStore from 'amo/store';
+import { selectReview } from 'amo/reducers/reviews';
 import { createInternalReview, setLatestReview } from 'amo/actions/reviews';
-import {
+import RatingManager, {
   RatingManagerBase,
-  RatingManagerWithI18n,
   mapDispatchToProps,
   mapStateToProps,
 } from 'amo/components/RatingManager';
 import {
+  dispatchClientMetadata,
+  dispatchSignInActions,
   fakeAddon,
   fakeReview,
-  signedInApiState,
 } from 'tests/unit/amo/helpers';
 import {
+  createFakeLocation,
   createStubErrorHandler,
-  createUserAccountResponse,
   fakeI18n,
-  createFakeLocation,
-  userAuthToken,
+  shallowUntilTarget,
 } from 'tests/unit/helpers';
 
 describe(__filename, () => {
-  let store;
-
-  beforeEach(() => {
-    store = createStore().store;
-  });
-
   function render(customProps = {}) {
     const props = {
       AddonReview: () => <div />,
       AuthenticateButton: () => <div />,
       ReportAbuseButton: () => <div />,
       addon: createInternalAddon(fakeAddon),
-      apiState: signedInApiState,
       errorHandler: createStubErrorHandler(),
+      i18n: fakeI18n(),
+      loadSavedReview: () => Promise.resolve(),
       location: createFakeLocation({ pathname: '/some/location/' }),
-      version: fakeAddon.current_version,
-      userId: 91234,
+      store: dispatchSignInActions().store,
       submitReview: () => Promise.resolve(),
-      loadSavedReview: () => Promise.resolve(),
-      store,
+      userId: 91234,
+      version: fakeAddon.current_version,
       ...customProps,
     };
 
-    const root = findRenderedComponentWithType(
-      renderIntoDocument(
-        <I18nProvider i18n={fakeI18n()}>
-          <Provider store={props.store}>
-            <RatingManagerWithI18n {...props} />
-          </Provider>
-        </I18nProvider>,
-      ),
-      RatingManagerBase,
+    return shallowUntilTarget(<RatingManager {...props} />, RatingManagerBase);
+  }
+
+  const createStoreWithLatestReview = ({
+    addon = fakeAddon,
```

Comment:
To make the intention of this more clear, it should be:

```js
addon = createInternalAddon(fakeAddon),
```

Reply:
oops, right